examples/README.md

```diff
@@ -0,0 +1,9 @@
+# BluePyEModel Examples
+
+This directory includes various examples demonstrating how to use BluePyEModel:
+
+* L5PC:  Provides a guide for setting up and running the E-Model building pipeline locally or on a Slurm cluster.
+
+* nexus: Demonstrates the use of BluePyEModel through the Nexus access point.
+
+* others: Contains various examples demonstrating different functionalities and use cases.
```

Comment:
It would be nice to have a description of all the subfolder examples with hyperlinks to their README for each one.  Please provide hyperlinks for the `L5PC` and `nexus` folders, too.

Reply:
done in one of the last commits